learning_wavelets/training_scripts/exact_recon_unet_training.py

```diff
@@ -80,7 +84,7 @@ def train_unet(
         steps_per_epoch=200,
         epochs=n_epochs,
         validation_data=im_ds_val,
-        validation_steps=1,
+        validation_steps=100,
```

Comment:
I had forgotten that the validation dataset had a batch size already.
Let's make this smaller then
```suggestion
        validation_steps=15,
```